CalibTracker/SiStripQuality/plugins/SiStripQualityHotStripIdentifier.cc

```diff
@@ -167,7 +167,8 @@ void SiStripQualityHotStripIdentifier::resetHistos() {
 void SiStripQualityHotStripIdentifier::bookHistos() {
   edm::LogInfo("SiStripQualityHotStripIdentifier") << " [SiStripQualityHotStripIdentifier::bookHistos] " << std::endl;
   char hname[1024];
-  for (const auto& it : SiStripDetInfoFileReader::read(fp_.fullPath()).getAllData()) {
+  const auto info = SiStripDetInfoFileReader::read(fp_.fullPath());
```

Comment:
Why not
```suggestion
  const auto& info = SiStripDetInfoFileReader::read(fp_.fullPath());
```
or even
```suggestion
  const SiStripDetInfo& info = SiStripDetInfoFileReader::read(fp_.fullPath());
```
?